README.md

```diff
@@ -85,13 +85,27 @@ A temporal dimension based on the ISO 8601 standard. The temporal reference syst
 | values     | \[string]       | If the dimension consists of an ordered list of specific values they can be listed here. The dates and/or times must be strings compliant to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601). |
 | step       | string\|null    | The space between the temporal instances as [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations), e.g. `P1D`. Use `null` for irregularly spaced steps. |
 
+### Vector Dimension Object
+
+A vector dimension that defines a spatial dimension based on geometries.
+
+| Field Name       | Type           | Description                                                  |
+| ---------------- | -------------- | ------------------------------------------------------------ |
+| type             | string         | **REQUIRED.** Type of the dimension, always `vector`.        |
+| axes             | \[string]      | Axes of the vector dimension as an ordered set of `x`, `y` and `z`. Defaults to `x` and `y`. |
+| description      | string         | Detailed multi-line description to explain the dimension. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| bbox             | \[number]      | **REQUIRED.** A single bounding box of the geometries as defined for [STAC Collections](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#spatial-extent-object), but not nested. |
+| values           | \[string\|[GeoJSON Object](https://tools.ietf.org/html/rfc7946)] | Optionally, a representation of the vectors. This can be a list of WKT strings, GeoJSON objects or identifiers. |
+| geometry_types   | \[[GeoJSON Types](https://www.rfc-editor.org/rfc/rfc7946#section-1.4)] | A set of geometry types. If not present, mixed geometry types must be assumed. |
+| reference_system | string\|number\|object | The spatial reference system for the data, specified as [numerical EPSG code](http://www.epsg-registry.org/), [WKT2 (ISO 19162) string](http://docs.opengeospatial.org/is/18-010r7/18-010r7.html) or [PROJJSON object](https://proj.org/specifications/projjson.html). Defaults to EPSG code 4326. |
+
```

Comment:
Instead of a number, it would be better to use identifiers such as `EPSG:4326`, as this allows for other authorities than EPSG (e.g., ESRI, OGC). A default that is better than `EPSG:4326` is `OGC:CRS84`; it points to the same datum (ensemble?) but makes explicit that coordinates are pairs of (longitude,latitude), where `EPSG:4326` _suggests_ that coordinates are pairs of (latitude,longitude) which in practice they almost always aren't. Using `OGC:CRS84` leaves that ambiguity behind us.

Reply:
Yeah, that ship has sailed in STAC. We either use EPSG code or PROJJSON or WKT2 for now...
We can say we default to another CRS, but we can then only encode it in PROJJSON or WKT2.